examples/DumpNTLMInfo.py

```diff
@@ -503,7 +503,7 @@ def DisplaySmbInfo(self):
 
         negotiation = connection.NegotiateSession()
         dialect = negotiation['DialectRevision']
-        secMode = negotiation['SecurityMode']
+        secMode = negotiation['SecurityMode'] if 'SecurityMode' in negotiation.fields.keys() else 0

```

Comment:
```suggestion
        secMode = negotiation['SecurityMode']
```

Reply:
Reverted to the original as will be handled in `_wrapper` function

Reply:
Fixed!